components/mosart/src/riverroute/RtmMod.F90

```diff
@@ -4212,7 +4215,7 @@ subroutine MOSART_init
   end if  ! endr >= begr
 
   ! retrieve the downstream channel attributes after some post-processing above
-  if (Tctl%RoutingMethod == 2 ) then       ! Use diffusion wave method in channel routing computation.
+  if (Tctl%RoutingMethod == DW ) then       ! Use diffusion wave method in channel routing computation.
```

Comment:
Are these mosart changes due to differences between the head of ocean discussions and e3sm/master?

Reply:
They shouldn't be.  I just updated E3SM-Ocean-Discussion `master` to match E3SM-Project's `master` and they're still here.

There are 7 odd commits listed in this PR that shouldn't be here.  @scalandr, if you can rebase once again onto E3SM-Project/E3SM/master (i.e. `origin/master`) and remove the first 7 commits at the top from the list, that should take care of it.  It should be an easy rebase because no conflicts are listed here on GitHub.